src/main/java/bio/terra/workspace/db/ResourceDao.java

```diff
@@ -250,12 +250,10 @@ public void createControlledResource(ControlledResource controlledResource)
         controlledResource.getResourceType(),
         controlledResource.getCloningInstructions(),
         controlledResource.attributesToJson(),
-        // TODO: add this to ControlledResource
-        Optional.of(AccessScopeType.ACCESS_SCOPE_SHARED),
-        Optional.of(ManagedByType.MANAGED_BY_USER),
+        Optional.ofNullable(controlledResource.getAccessScope()),
+        Optional.ofNullable(controlledResource.getManagedBy()),
         // TODO: add associated app to ControlledResource
         Optional.empty(),
-        // TODO: rename this
         controlledResource.getAssignedUser());
   }
```

Comment:
very optional: consider changing the helper `storeResource` to take a `WsmResource` for the common fields to greatly cut down the arguments ane make clearer it's for storing WsmResource subclasses.